kavaplugin/src/main/java/com/kaltura/playkit/plugins/kava/OptionalParams.java

```diff
@@ -13,35 +13,35 @@
     OptionalParams(KavaAnalyticsConfig config) {
         optionalParams = new LinkedHashMap<>();
 
-        if (config.hasPlaybackContext()) {
+        if (config.hasPlaybackContext() && config.getPlaybackContext() != null) {
```

Comment:
No need to ask config.hasPlaybackContext()  AND  config.getPlaybackContext() != null -- it's the same.